ultraviolet/shared/src/main/scala/ultraviolet/macros/ShaderProgramValidation.scala

```diff
@@ -151,6 +152,32 @@ object ShaderProgramValidation:
             Val(id, validate(level + 1, knownRefs)(value), typeOf)
       else Val(id, validate(level + 1, knownRefs)(value), typeOf)
 
+    case MultiStatements(typeOf, vals) =>
+      val vs = vals.map {
+        case Val(id, value, typeOf_) =>
+          // TODO: typeof
```

Comment:
Done?

Reply:
Fixed in https://github.com/PurpleKingdomGames/ultraviolet/pull/142/commits/a25ee888a2006d59ab60bc64b9cca815729f5203